source/includes/cli-reference/_projectfile.html.md

```diff
@@ -0,0 +1,63 @@
+# Projects and Config Files
+
+The Bonsai command-line interface works with three files.  A `.bonsai` file in
+your user directory, a project file in the directory containing the Inkling 
+files and simulator configuration for a BRAIN, and a `.brains` file that links
+a project file to a BRAIN in the same directory as the project file.
+
+## `.bonsai`
+
+The `.bonsai` file stores your username and an access token for access to the

```

Comment:
Do we not want to show a small code sample of what belongs in this file? Even with dummy values in it?